aries-site/src/pages/extend/developer-guidance.js

```diff
@@ -0,0 +1,128 @@
+      />
+      <ContentSection>
+        <Subsection name={title} level={1} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+        </Subsection>
+        <Subsection name="Getting started">
+          <SubsectionText>
+            We're excited you're using the HPE Design System! Below are a set of
+            resources that will help you get your application set-up with
+            Grommet and the HPE theme. If you don't find what you're looking
+            for, please reach out in the #hpe-design-system channel on{' '}
+            <Anchor
+              label="Slack"
+              href="https://grommet.slack.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            />
+            .
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Preferred environment" level={3}>
+          <SubsectionText>
+            The HPE Design System is primarily focused on user interfaces
+            developed with ReactJS and Grommet. We encourage teams to use
+            ReactJS and Grommet if possible because there are more resources,
+            examples, and community help for this environment.
+          </SubsectionText>
+          <SubsectionText>
+            Questions and feedback should be routed through the
+            #hpe-design-system channel on{' '}
+            <Anchor
+              label="Slack"
+              href="https://grommet.slack.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            />
+            .
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="ReactJS and Grommet starter resources" level={3}>
+          <SubsectionText>
+            If you are new to ReactJS, the{' '}
+            <Anchor
+              label="React Tutorial"
+              href="https://reactjs.org/tutorial/tutorial.html"
+              target="_blank"
+              rel="noopener noreferrer"
+            />{' '}
+            is a good place to start.
+          </SubsectionText>
+          <SubsectionText>
+            If you are already familiar with ReactJS but are unfamiliar with
+            Grommet, check out the{' '}
+            <Anchor
+              label="Getting Started with Grommet"
+              href="https://v2.grommet.io/starter"
+              target="_blank"
+              rel="noopener noreferrer"
+            />{' '}
+            guide.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Applying the HPE theme" level={3}>
+          <SubsectionText>
+            Once your project is set up with ReactJS and Grommet, make sure you
+            have the latest{' '}
+            <Anchor
+              href="https://github.com/grommet/grommet-theme-hpe"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <code>grommet-theme-hpe</code>
```

Comment:
Mixing anchor and code styling is a bit confusing, especially when 'npm' and 'yarn' follow and aren't clickable.
Soft opinion.

```suggestion
              grommet-theme-hpe
```

![Screen Shot 2020-04-27 at 2 31 13 PM](https://user-images.githubusercontent.com/1756948/80417902-c4b52f80-8893-11ea-885d-3e07be3bc81e.png)